app/main.py

```diff
@@ -1,8 +1,9 @@
 """Main app."""
 
-import os
 import warnings
 from contextlib import asynccontextmanager
+
+# from functools import lru_cache
```

Comment:
Is this leftover? If not (e.g. for future use), please add a TODO flag and explain when we want to uncomment